WORKSPACE

```diff
@@ -29,3 +31,55 @@ com_google_gapic_generator_cpp_repositories()
 load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
+
+# WARNING: The java and go segments below are a hack to get
```

Comment:
Is this temporary?  Or are we planning to ask users to always install go and java to compile their C++ code?

If it is temporary, do you mind adding a bug in GitHub and a `TODO(#bug-number)` reminding folks to remove this?

Reply:
I don't have a good idea of how temporary this solution is or what should be done to fix it. @vam-google was a big help in deriving these workarounds and can answer those questions better. I'll file the bug and update the comment.

Reply:
Tracked by #35

Reply:
This is currently work in progress (an item I'm currently working on), will be ready very soon (days from now).